docs/codingconventions.adoc

```diff
@@ -21,8 +31,75 @@ It is not an exhaustive list and it will be updated as we go. In the mean time,
 - Nested modules should exist under the `modules/` folder
 - Examples of using the module should exist under the `examples/` folder and have a README to explain the goal and usage of the example
 
-== Case type and Names
+== Documentation format
+
+We use https://asciidoc.org/[AsciiDoc] with the `*.adoc` file extension. The only exception is for the README files that need to be displayed on the Terraform Registry.
+
+[NOTE]
+====
+As GitHub renders AsciiDoc files using https://asciidoctor.org/[AsciiDoctor], it is the current reference regarding AsciiDoc syntax we use in the documentation.
+====
+
+Terraform Registry currently do not support AsciiDoc,so documentation content such as `README` files must be in Markdown format.
+
+=== Internal links
+
+For internal links to AsciiDoc file (repo documentation content), we use relative links with the `xref:` macro.
+
+```
+xref:CONTRIBUTING.adoc[CONTRIBUTING]
+```
+
+For internal links to non-AsciiDoc file (e.g: README files), we use relative links with the `link:` macro.
+
+```
+link:examples/README.md[examples README]
+```
+
+=== External links
+
+For external links, we use https://docs.asciidoctor.org/asciidoc/latest/macros/autolinks/[autolinks].
+
+[TIP]
+====
+If a link is used mutiple times, prefer to add the link in the header with a reference in the format `:uri-xxx:`. This will facilitate updates in link destination is to be modified.
```

Comment:
Try: "Links that are used multiple times must be defined in the header using the `:uri-xxx` reference format. This makes any future updates to the URI much simpler."